package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
For now we should remove it. We will be introducing unit tests very soon as then we can update `package.json` and `ci.sh` (for CircleCI to run our tests as well).